src/mentorship/ml/cv/util.py

```diff
@@ -1,7 +1,9 @@
 import numpy as np
+from matplotlib import pyplot as plt
 
 
 def print_cv_test_scores(scores):
+    scores_RMSLE = []
```

Comment:
`scores_RMSLE` is not a genric variable name inside the generic function. I would suggest you generalizing your function if you want:
```suggestion
def display_cv_test_scores(scores, plot_scores=None):
    metrics_to_plot = {}
    # if metric_name in plot_scores: ....
```